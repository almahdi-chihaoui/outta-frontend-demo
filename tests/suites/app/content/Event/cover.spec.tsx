import React from "react";
import renderer from "react-test-renderer";

import Cover from "../../../../../src/containers/app/content/Event/cover";

describe('Cover Component', () => {
  let component;

  beforeEach(() => {
    const props = {
      title: 'bla',
      img: 'https://www.pro-sky.com/images/en/news/news_header_webseite_-49-_342_big.jpg',
      description: 'blabla',
      isLoading: false,
    }
    component = renderer.create(
      <Cover {...props}></Cover>
    );
  });

  it('should render', () => {
    expect(component.toJSON()).toMatchSnapshot();
  });
});