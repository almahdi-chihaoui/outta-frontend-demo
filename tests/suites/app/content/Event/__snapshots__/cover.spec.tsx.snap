// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cover Component should render 1`] = `
<div
  className="MuiPaper-root makeStyles-cover-1 MuiPaper-elevation1"
>
  <img
    alt=""
    className="makeStyles-img-3"
    src="https://www.pro-sky.com/images/en/news/news_header_webseite_-49-_342_big.jpg"
  />
  <div
    className="makeStyles-desc-2"
  >
    <h4
      className="MuiTypography-root makeStyles-title-4 MuiTypography-h4"
    >
      bla
    </h4>
    <p
      className="MuiTypography-root MuiTypography-body1"
    >
      blabla
    </p>
  </div>
</div>
`;
